public/global.css

```diff
@@ -58,6 +58,10 @@ input:disabled {
 	background: #ccc;
 }
 
+::-webkit-scrollbar-corner {
+	background-color: #fff;
```

Comment:
Dark mode et light mode ?

Reply:
For dark mode it's in the end of file: 

::-webkit-scrollbar-corner {
	background-color: #444;
}